kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/util/KuraBaseModel.java

```diff
@@ -20,6 +20,8 @@
      */
     private static final long serialVersionUID = -2502956403624362215L;
 
+    private transient boolean unescape = false;
```

Comment:
I'd prefer ``unsecaped``. The initializer is not needed.
Update end date in the copyright header.